src/py-opentimelineio/opentimelineio/console/otiotool.py

```diff
@@ -369,6 +385,23 @@ def parse_arguments():
     )
 
     # Output...
+    parser.add_argument(
+        "--downgrade",
+        type=str,
+        metavar='FAMILY:VERSION',
+        help="""Downgrade OTIO schema. Only relevant when --output is used
+        to output an OTIO file. FAMILY:VERSION specifies which schema family
+        and version to use. If FAMILY: is omitted, the default OTIO_CORE: is
+        used. For example `--downgrade OTIO_CORE:0.14.0` is equivalent to
+        `--downgrade 0.14.0`. See the OpenTimelineIO documentation for
+        OTIO_DEFAULT_TARGET_VERSION_FAMILY_LABEL for details."""
```

Comment:
Would it be a good idea to link to https://opentimelineio.readthedocs.io/en/latest/tutorials/otio-env-variables.html#term-OTIO_DEFAULT_TARGET_VERSION_FAMILY_LABEL and or https://opentimelineio.readthedocs.io/en/latest/tutorials/versioning-schemas.html?

Reply:
Linked.